mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster.go

```diff
@@ -1638,7 +1637,6 @@ by Dataproc`,
 										},
 									},
 									"node_group_id": {
-										Optional:    true,
 										Computed:    true,
 										Type:        schema.TypeString,
 										Description: `A node group ID. Generated if not specified. The ID must contain only letters (a-z, A-Z), numbers (0-9), underscores (_), and hyphens (-). Cannot begin or end with underscore or hyphen. Must consist of from 3 to 33 characters.`,
```

Comment:
this description claims the field can be specified -- as does the API documentation here https://cloud.google.com/dataproc/docs/reference/rest/v1/ClusterConfig#AuxiliaryNodeGroup, and is included even as suggested to be declared within the instructions at https://cloud.google.com/dataproc/docs/guides/node-groups/dataproc-driver-node-groups#dataproc_create_driver_node_group-drest

is this incorrect or misleading? It seems like this field *should* be optional+computed from the wording.

Reply:
yes, but current priority from dataproc product team is to support computed only.
auxiliary node groups will always be created, instead of using the existing ones for now.

@ahzaz@google.com could you please confirm and suppliment?